maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleProjectTests.java

```diff
@@ -589,49 +589,125 @@ public void testProjectTemplateFormField() throws Exception
     }
 
     @Test
-    public void testProjectTemplateFragment() throws Exception
```

Comment:
You removed the testProjectTemplateFragment, can you add it back?

Reply:
Ok, got it. ProjectTemplateFragment Junit test is written in  MavenModuleFragmentProjectTests. And I will also add this test back in MavenModuleProjectTests.